prime-router/src/main/kotlin/azure/CheckFunction.kt

```diff
@@ -17,8 +17,6 @@ import gov.cdc.prime.router.azure.db.enums.SettingType
 import gov.cdc.prime.router.common.BaseEngine
 import gov.cdc.prime.router.common.JacksonMapperUtilities
 import gov.cdc.prime.router.credentials.RestCredential
-import gov.cdc.prime.router.tokens.AuthenticatedClaims
```

Comment:
@arnejduranovic , this change is not correct.   Some code that used these classes was erroneously commented out.   I asked Ott to correct that, and he has merged that fix into master, and now these two imports are actually needed.    So you can remove these changes to CheckFunction from your PR.